src/vcs/git.rs

```diff
@@ -269,6 +269,39 @@ impl<'repo> Repository {
         }
     }
 
+    pub(crate) fn branch_contains(&self, commit: &Oid) -> Result<Vec<Branch>, Error> {
```

Comment:
I expected this function to be a `(branch, rev) -> bool` function, given its name, ie. tell me if this branch contains this rev. I think we should try to find a better name for it, or possibly move it to the `Commit`? Brainstorming:
```
commit.branches() -> Vec<Branch>
repo.branches_with(rev) -> Vec<Branch>
repo.commit_branches(commit) -> Vec<Branch>
```

Reply:
It can't really go on a `Commit` because a `Commit` doesn't know about search history.

Changing it to accept revisions could definitely, given it resolves to a `Commit` of course :thinking:

Reply:
yeah then I guess `Repository::commit_branches` or `revision_branches` might be clearer?

Reply:
Yup makes sense!